analytics/analytics.go

```diff
@@ -62,9 +66,31 @@ func DetectCI(isEarthlyCIRunner bool) (string, bool) {
 		}
 	}
 
+	// detached head is often used in CIs. This is another heuristic.
```

Comment:
I often work in detached head when reviewing PRs, I would take this with a grain of salt.

Reply:
Indeed. Let's see what the data looks like.